### PR TITLE
Release v0.18.0

### PR DESCRIPTION
**Pull request by Hugoo**
### ⚠ BREAKING CHANGES

- new encoding for static value types (not arrays `[]`) (#288)
- change `ArrayLength` value from `uint256` to `uint128` (#287)

### Features

- add `checkPermissions` function ([17d2258](https://github.com/ERC725Alliance/erc725.js/commit/17d225843c236951ef1515a0ff91095b5ef27cd3))
- Add new feature "Decode Mapping Key" ([8c1f1fc](https://github.com/ERC725Alliance/erc725.js/commit/8c1f1fcfb15fa43d1d3934b0b15f09d47902bb41))

### Bug Fixes

- decode any uint256 as number not string ([#289](https://github.com/ERC725Alliance/erc725.js/issues/289)) ([37203f1](https://github.com/ERC725Alliance/erc725.js/commit/37203f14d313a0caff75724dc74175c741c1b540))
- variable naming ([44b4785](https://github.com/ERC725Alliance/erc725.js/commit/44b47851ed63b817edc21c63655d67bac13a7e7f))

### improvement

- change `ArrayLength` value from `uint256` to `uint128` ([#287](https://github.com/ERC725Alliance/erc725.js/issues/287)) ([c95ee8a](https://github.com/ERC725Alliance/erc725.js/commit/c95ee8a53bf25bcf47777054af27cae1fbad8b2f))
- new encoding for static value types (not arrays `[]`) ([#288](https://github.com/ERC725Alliance/erc725.js/issues/288)) ([f0b04da](https://github.com/ERC725Alliance/erc725.js/commit/f0b04daa57a281c537a8f28594439573188f0dce))

### CODE REVIEW

**Review by CallumGrindle — APPROVED** [review]
🎊

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+## [0.18.0](https://github.com/ERC725Alliance/erc725.js/compare/v0.17.2...v0.18.0) (2023-08-03)
+
+### ⚠ BREAKING CHANGES
+
+- new encoding for static value types (not arrays `[]`) (#288)
+- change `ArrayLength` value from `uint256` to `uint128` (#287)
+
+### Features
+
+- add encoding / decoding for `bytes[CompactBytesArray]` ([#261](https://github.com/ERC725Alliance/erc725.js/issues/261)) ([8d3e4e9](https://github.com/ERC725Alliance/erc725.js/commit/8d3e4e994957dbf1f54eef4572b072bc308c61bc))
```
Comment (CJ42) [inline CHANGELOG.md:14]: This was part of the previous release

image

**Review by CJ42 — CHANGES_REQUESTED** [review]
There seem to be commits and PRs reference that are very old, for features/fixes that have already been released.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+- add `BitArray` in valueContent encoding map ([3498502](https://github.com/ERC725Alliance/erc725.js/commit/3498502da0b531559e557fc4a9e6c2851b480807))
+- add checkPermissions function ([17d2258](https://github.com/ERC725Alliance/erc725.js/commit/17d225843c236951ef1515a0ff91095b5ef27cd3))
+- Add new feature "Decode Mapping Key" ([8c1f1fc](https://github.com/ERC725Alliance/erc725.js/commit/8c1f1fcfb15fa43d1d3934b0b15f09d47902bb41))
+- add supportsInterface ([#243](https://github.com/ERC725Alliance/erc725.js/issues/243)) ([a2b0828](https://github.com/ERC725Alliance/erc725.js/commit/a2b08288e9aaede8a1a3307c1371672eb7b50969))
```
[the remark this answers is not in the document]
Comment (Hugoo) [reply CHANGELOG.md]: Good catch!

File: CHANGELOG.md (hunk 1 above)
Comment (Hugoo) [reply CHANGELOG.md]: good catch - we need to fix that release ci

**Review by CJ42 — APPROVED** [review]

**Pull request merged**